### PR TITLE
Bug Fix:Fix Data Update feed_source_url Script

### PR DESCRIPTION
**Pull request by mstruve**
## What type of PR is this? (check all applicable)
- [x] Bug Fix

## Description
This script is not quite right and needs a little fixing to ensure Elasticsearch data is removed properly. We dont need this anymore but I want to make sure the script is correct in case anyone else needs it and/or for people who might refer to the code to base their own code off of. 

alt_text

### CODE REVIEW

File: lib/data_update_scripts/20200904132553_remove_draft_articles_with_duplicate_feed_source_url.rb [hunk 1]
```diff
@@ -6,7 +6,7 @@ def run
       # This statement deletes all draft articles in excess found to be duplicate over feed_source_url,
       # excluding those whose body_markdown is different from the other duplicate occurrences
       result_same_body = ActiveRecord::Base.connection.execute(
-        <<~SQL,
+        <<~SQL.squish,
```
Comment (mstruve) [inline lib/data_update_scripts/20200904132553_remove_draft_articles_with_duplicate_feed_source_url.rb:9]: Rubocop fix you can learn more about it in this PR: https://github.com/forem/forem/pull/10237

Comment (juliannatetreault) [reply lib/data_update_scripts/20200904132553_remove_draft_articles_with_duplicate_feed_source_url.rb:9]: TIL `.squish` -- love it!

**Review by juliannatetreault — APPROVED** [review]
TIL about `.squish` and the `searchable` concern. 😄 I don't see anything wrong with these changes, so 🚀 !

**Review by maestromac — APPROVED** [review]
LGTM!

**Pull request merged**